ballista/core/src/execution_plans/distributed_query.rs

```diff
@@ -274,10 +275,11 @@ async fn execute_query(
     session_id: String,
     query: ExecuteQueryParams,
     max_message_size: usize,
+    config: BallistaConfig,
```

Comment:
we already have gpc_max_message_size as a parameter can we pass other needed values as parameters rather than passing config